docs/operations/rwx.md

```diff
@@ -1,136 +1,431 @@
+
+ ID                      aad1e4ab-23d2-4a45-9a9b-0305885997ff
+ Name                    pvc-41f429ff-3fa9-4361-b7bc-c55f869499ae
+ Description
+ AttachedOn              aks-default-15645363-vmss000000 (1c9fe8f8-7c33-44b0-802d-875ff4c53fdd)
+ Attachment Type         nfs
+ NFS
+   Service Endpoint      10.224.0.4:25918
+   Exports:
+   - ID                  1
+     Path                /
+     Pseudo Path         /
+     ACLs
+     - Identity Type     hostname
+       Identity Matcher  *
+       Squash            all
+       Squash UID        0
+       Squash GUID       0
+ Namespace               ondat-files (97e1ffeb-34e3-4f18-8ef1-5494f34fe9cd)
+ Labels                  csi.storage.k8s.io/pv/name=pvc-41f429ff-3fa9-4361-b7bc-c55f869499ae,
+                         csi.storage.k8s.io/pvc/name=ondat-files,
+                         csi.storage.k8s.io/pvc/namespace=ondat-files,
+                         storageos.com/nfs/mount-endpoint=10.0.133.137:2049,
+                         storageos.com/nocompress=true
+ Filesystem              ext4
+ Size                    5.0 GiB (5368709120 bytes)
+ Version                 NQ
+ Created at              2022-09-08T12:54:49Z (2 hours ago)
+ Updated at              2022-09-08T14:29:18Z (28 minutes ago)
+
+ Master:
+   ID                    67cae629-7797-444a-828d-22d5c22d3535
+   Node                  aks-default-15645363-vmss000000 (1c9fe8f8-7c33-44b0-802d-875ff4c53fdd)
+   Health                online
+ ```
+
+1. Notice under `Attachment Type         nfs`, there detailed information about the RWX volume is returned.
+
+ ```yaml
+ # truncated output..
+ Attachment Type         nfs
+ NFS
+   Service Endpoint      10.224.0.4:25918
+   Exports:
+   - ID                  1
+     Path                /
+     Pseudo Path         /
+     ACLs
+     - Identity Type     hostname
+       Identity Matcher  *
+       Squash            all
+       Squash UID        0
+       Squash GUID       0
+ # truncated ouput...
+ ```
+
+## NFS Squash Mode
+
+> 💡 This feature is available in release  `v2.8.0`  or greater.
+
+As part of the `v2.8.0` Ondat release, a change was made so that users can [configure the squash mode](https://linux.die.net/man/5/exports) for the NFS service that is used to provision RWX shares.
+
+- Historically, all shares were exported with a `Squash = All`  mode of operation. This was requested by most customers as the idea of identity in a container based deployment is very abstract.
+- There is now a Ondat volume [Feature label](/docs/concepts/labels/) that can applied to make this setting configurable. End users can now adjust the squash mode using the following label >> `storageos.com/nfs-squash: $APPLY_SQUASH_MODE_VALUE` when provisioning RWX volumes. Below are the list of different modes that users can apply (`all` is the default mode is the label is not applied to a `PersistentVolumeClaim`)
```

Comment:
that can **be** applied to ...

Reply:
Suggested: (`all` is the default mode **and** the label is not applied to a `PersistentVolumeClaim`)

Reply:
fixed